src/Model/Operation/AssetClaimFeesOperation.php

```diff
@@ -113,7 +114,10 @@ public function toBytes(): string
             $this->getUia()->toBytes(),
             $this->getDct()->toBytes(),
             // TODO: Extensions Array
-            $this->getExtensions() ? '01' : '00'
+            $this->getExtensions() ?
+                VarInt::encodeDecToHex(sizeof($this->getExtensions()))
+                . '' // TODO array_map for every element in array
```

Comment:
unresolved todo

Reply:
Unknown structure of array, can't fix now.